### DISCUSSION TITLE
aws 亚马逊访问alist的网盘都会报错

### DISCUSSION THREAD

**Discussion by HaitaoShi** [AlistGo/alist#3050] [General]
failed get objs: failed to list objs: Get "https://webapi.115.com/files?aid=1&asc=1&cid=0&fc_mix=0&format=json&limit=56&natsort=0&o=user_ptime&offset=0&record_open_time=1&show_dir=1&snap=0": dial tcp ************3:443: i/o timeout

请问大神这是由于网卡顿还是bug？本地的没问题，但不能挂网盘。。。谢谢

**Comment by NovaShen555** [comment]
没太看懂你想讲啥，aws访问？那你给个复现链接吧

**Reply by HaitaoShi** [reply]
我解决了，用的oracle的服务器。aws不知道为什么。
